pydra/engine/helpers.py

```diff
@@ -544,3 +508,37 @@ def get_available_cpus():
 
     # Last resort
     return os.cpu_count()
+
+
+def load_and_run(task_pkl, ind=None, rerun=False, **kwargs):
+    """
+     loading a task from a pickle file, settings proper input
+     and running the task
+     """
+    task = load_task(task_pkl=task_pkl, ind=ind)
+    task._run(rerun=rerun, **kwargs)
```

Comment:
i would put this in a try - except clause, since things could fail for various reasons and then update the result file appropriately whether things crashed or not.

Reply:
ok

Reply:
which result file you have in mind?

Reply:
the result file created by the task. in case the task crashes then a result file created by the code that traps that exception.

Reply:
so you want to write an error into `_result.pklz`? not sure if I follow

Reply:
i've added a simple crashfile